sdkconfig.defaults

```diff
@@ -61,25 +91,28 @@ CONFIG_PARTITION_TABLE_FILENAME="partitions-16MB.csv"
 CONFIG_APP_OFFSET=0x10000
 
 #
-# Connectivity
-#
-CONFIG_WIFI_USE=y
-CONFIG_WIFI_SSID="SHA2017-insecure"
-CONFIG_WIFI_PASSWORD=""
-
-#
-# MicroPython hacks
+# Compiler options
 #
-CONFIG_SUPPORT_STATIC_ALLOCATION=y
-CONFIG_ENABLE_STATIC_TASK_CLEAN_UP_HOOK=y
-CONFIG_FREERTOS_THREAD_LOCAL_STORAGE_POINTERS=2
-CONFIG_ESP32_DEEP_SLEEP_WAKEUP_DELAY=2000
 CONFIG_OPTIMIZATION_LEVEL_DEBUG=y
 # CONFIG_OPTIMIZATION_LEVEL_RELEASE is not set
+CONFIG_OPTIMIZATION_ASSERTIONS_ENABLED=y
```

Comment:
asserts are only used in rare situations which we didn't expect. I prefer the full assertion errors. When someone triggers an assertion, we know where it comes from.